### DISCUSSION TITLE
Adding a category for Insights

### DISCUSSION THREAD

**Discussion by dduzgun-security** [community/community#55741] [Other Feature Feedback, Questions, & Ideas]
## Suggestion
Adding a category for `Insights` on the discussion feeds. 👇 

<img width="285" alt="image" src="https://github.com/community/community/assets/59659739/b7009e63-3b36-4bb5-8195-6d12f2943a15">

**Comment by ghostinhershell** [comment]
Hey @dduzgun-security, 

Thanks so much for writing in and thanks so much for the suggestion. Can you tell me a bit more about what the Insights category would consist of?

**Reply by dduzgun-security** [reply]
👋 @ghostinhershell,
I was thinking about this as a category for:
- suggestions/discussions of new insights overviews (ex: Security Overview, Statistics, Custom Dashboards, ...) ideas
- sharing and discussing around tips on queries for Dependencies Insights
- suggestions on enhancing queries (ex: adding a created-at filter)

Overall for feedbacks and tips across the Insight functionality that GitHub offers that may benefit the community.
